ui/DesignSystem/Component/AdditionalActionsDropdown.svelte

```diff
@@ -53,8 +53,15 @@
     outline-style: none;
   }
 
+  button :global(svg) {
```

Comment:
We should use a unique classname for the button in this component, something like `.additional-actions-dropdown-button`, otherwise this global style might do something unexpected in case we have a `<button>` tag with an `<svg>` as child somewhere else in the code.